app/Models/Cts/ValidationPosition.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models\Cts;
+
+use Illuminate\Database\Eloquent\Model;
+
+class ValidationPosition extends Model
+{
+    protected $connection = 'cts';
+    protected $table = 'validations_p';
+    protected $guarded = [];
```

Comment:
Do you want to use fillable none instead? We shouldn't theoretically modifying anything.

Reply:
Good shout - resolved in commit b94d9250